### PR TITLE
More than one organization unit for user

### LINKED ISSUE #958: More than one organization unit for user
With percentage of each organization. Should be editable in UserUpdateForm. Don't forget add multiple organizations in acceptUserRequest when creating newUser.

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #958

## PR includes

-   [x] Feature

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -122,7 +122,8 @@ model User {
 
   mailingSettings MailingSettings[]
 
-  workStartDate DateTime?
+  workStartDate        DateTime?
+  supplementalPosition SupplementalPosition[]
```
Comment (tsumo) [inline prisma/schema.prisma:126]: `supplementalPositions`, since it's an array

File: prisma/schema.prisma [hunk 2]
```diff
@@ -184,6 +185,8 @@ model UserCreationRequest {
   jobId String? @unique
   job   Job?    @relation(fields: [jobId], references: [id])
 
+  supplementalPosition SupplementalPosition[]
```
Comment (tsumo) [inline prisma/schema.prisma:188]: I think we have to store this data like we do with services - as `Json`

File: prisma/schema.prisma [hunk 3]
```diff
@@ -335,6 +338,7 @@ model OrganizationUnit {
   oldOrganizationUnit  ScheduledDeactivation[] @relation("oldOrganizationUnit")
   newOrganizationUnit  ScheduledDeactivation[] @relation("newOrganizationUnit")
   mailingSettings      MailingSettings[]
+  supplementalPosition SupplementalPosition[]
```
Comment (tsumo) [inline prisma/schema.prisma:341]: `supplementalPositions`

File: prisma/schema.prisma [hunk 4]
```diff
@@ -671,3 +675,17 @@ model OrganizationDomain {
...
 }
+
+model SupplementalPosition {
+  id String @id @default(cuid())
+
+  organizationUnit   OrganizationUnit @relation(fields: [organizationUnitId], references: [id])
+  organizationUnitId String
+
+  user       User?   @relation(fields: [userId], references: [id])
+  userId     String?
```
Comment (tsumo) [inline prisma/schema.prisma:686]: userId should not be optional

File: src/components/CreateUserCreationRequestInternalEmployeeForm/CreateUserCreationRequestInternalEmployeeForm.i18n/ru.json [hunk 5]
```diff
@@ -31,5 +31,6 @@
     "Cancel": "Отмена",
     "Comment:": "Комменарий",
     "Enter your comment": "Введите комментарий",
-    "User with login already exist": "Уже есть пользователь с таким логином"
+    "User with login already exist": "Уже есть пользователь с таким логином",
+    "Supplemental position:": "Дополнительные организации:"
```
Comment (tsumo) [inline src/components/CreateUserCreationRequestInternalEmployeeForm/CreateUserCreationRequestInternalEmployeeForm.i18n/ru.json:35]: positions

File: src/modules/userCreationRequestSchemas.ts [hunk 6]
```diff
@@ -39,6 +39,7 @@ export const createUserCreationRequestInternalEmployeeSchema = createUserCreatio
     creationCause: z.string(),
     unitId: z.string().min(1, { message: tr('Minimum {min} symbols', { min: 1 }) }),
     date: z.date(),
+    supplementalPosition: z.array(z.object({ organizationUnitId: z.string(), percentage: z.number() })).optional(),
```
Comment (tsumo) [inline src/modules/userCreationRequestSchemas.ts:42]: positions

**Review by tsumo — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 4 above)
Comment (nyet-ty) [reply prisma/schema.prisma:686]: What if we need supplemental positions for user request?

Comment (nyet-ty) [reply prisma/schema.prisma:686]: Or we can store in user request it in Json and create SupplementalPosition when creating user?

File: src/components/AddSupplementalPosition/AddSupplementalPosition.tsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+                            type="button"
+                            outline
+                            onClick={inLineFormSubmit}
+                        />
+                    </div>
+                ),
+                <InlineTrigger
+                    text={tr('Add supplemental position')}
+                    icon={<IconPlusCircleSolid size="s" />}
+                    onClick={() => formVisibility.setTrue()}
```
Comment (tsumo) [inline src/components/AddSupplementalPosition/AddSupplementalPosition.tsx:72]: `onClick={formVisibility.setTrue}`

File: src/components/SupplementalPositionItem/SupplementalPositionItem.tsx [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+interface SupplementalPositionItemProps {
+    supplementalPosition: AddSupplementalPositionType;
+    removeSupplementalPosition: (id: string) => void;
+}
+
+export const SupplementalPositionItem = ({
+    supplementalPosition,
+    removeSupplementalPosition,
+}: SupplementalPositionItemProps) => (
+    <div className={s.Base}>
```
Comment (tsumo) [inline src/components/SupplementalPositionItem/SupplementalPositionItem.tsx:18]: Maybe we can use Badge from bricks here?

File: src/modules/historyEventTypes.ts [hunk 9]
```diff
@@ -60,6 +60,7 @@ interface HistoryEventsData {
             buddyLogin?: string;
             coordinatorLogin?: string;
             recruiterLogin?: string;
+            supplementalPositions?: string;
```
Comment (tsumo) [inline src/modules/historyEventTypes.ts:63]: We can store array of objects here, no problem. Maybe we even extract analytics from them later

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/AddSupplementalPosition/AddSupplementalPosition.tsx (hunk 7 above)
Comment (tsumo) [reply src/components/AddSupplementalPosition/AddSupplementalPosition.tsx:72]: Just one suggestion is left

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/AddSupplementalPosition/AddSupplementalPosition.tsx (hunk 7 above)
Comment (nyet-ty) [reply src/components/AddSupplementalPosition/AddSupplementalPosition.tsx]: idk how did I missed it:/

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**